src/copaw/app/runner/runner.py

```
@@ -150,6 +150,31 @@ async def query_handler(
+                            msgs if isinstance(msgs, list) else [msgs]
+                        )
+                        if hasattr(m, "id")
+                    ]
+                    if msg_ids:
+                        await agent.memory.delete(msg_ids)
+                        logger.info(
+                            "Removed %d failed msg(s) from memory: %s",
+                            len(msg_ids),
+                            msg_ids,
+                        )
+                except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)